src/RoveComm/RoveCommPacket.h

```diff
@@ -0,0 +1,76 @@
+/******************************************************************************
+ * @brief RoveComm Packet Definition.
+ *
+ * @file RoveCommPacket.h
+ * @author Eli Byrd (edbgkk@mst.edu)
+ * @date 2024-02-05
+ *
+ * @copyright Copyright Mars Rover Design Team 2024 - All Rights Reserved
+ ******************************************************************************/
+
+#ifndef ROVECOMM_PACKET_H
+#define ROVECOMM_PACKET_H
+
+#include "./RoveCommConsts.h"
+#include "./RoveCommManifest.h"
+
+#include <cstdint>
```

Comment:
Put implicit include in 
/// \cond
#include <array>

/// \endcond
to avoid them being included in doxygen graph.